src/Dynamo.All.sln

```diff
@@ -512,7 +512,6 @@ Global
 		{F7FD9395-35D5-474D-8AD1-9904817E70D3}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{F7FD9395-35D5-474D-8AD1-9904817E70D3}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{F7FD9395-35D5-474D-8AD1-9904817E70D3}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{F7FD9395-35D5-474D-8AD1-9904817E70D3}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
please try to revert this change - I don't think there should be any modifications to this .sln right?

Reply:
I already reverted it.